skymap_scanner/server/start_scan.py

```diff
@@ -744,7 +808,7 @@ async def serve_scan_iteration(
                 except DuplicatePixelRecoException as e:
                     logging.error(e)
                 # if we've got all the pixrecos, no need to wait for queue's timeout
-                if len(col.pixreco_ids_received) == n_pixreco:
+                if len(col.pixreco_ids_sent-col.pixreco_ids_received) == 0:
```

Comment:
you can use truthy set-logic here: `if A - B:`

Reply:
or even `if A == B`, yeah?

Reply:
implemented `A==B`